packages/devtools_app/lib/src/shared/console/eval/diagnostics_node.dart

```diff
@@ -2,19 +2,18 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
This also isn't a console thing. If you want to move it, it should go in shared not console.